cmd/podman/common/create.go

```diff
@@ -158,7 +158,7 @@ func GetCreateFlags(cf *ContainerCLIOpts) *pflag.FlagSet {
 	)
 	createFlags.StringArrayVarP(
 		&cf.env,
```

Comment:
Remove the autosetting,  And check if the user actually did --env.

Reply:
fixed